circle/docker-development-image.sh

```diff
@@ -17,15 +17,30 @@
 CIRCLE_CI_FUNCTIONS_URL=${CIRCLE_CI_FUNCTIONS_URL:-https://raw.githubusercontent.com/bitnami/test-infra/master/circle/functions}
```

Comment:
Could you please updated this line so that it points to `https://raw.githubusercontent.com/tompizmor/test-infra/centos-poc/circle/functions`